Lib/sre_parse.py

```diff
@@ -72,7 +72,7 @@ class Verbose(Exception):
     pass
 
 class Pattern:
-    # master pattern object.  keeps track of global attributes
+    # main pattern object.  keeps track of global attributes
```

Comment:
Actually both wordings are wrong. This class doesn't represent a pattern object in any sense. I'll open a separate issue for this.